RELEASE.md

```diff
@@ -0,0 +1,39 @@
+# Strimzi Kafka OAuth release process
+
+This document describes the steps needed for release of Strimzi Kafka OAuth library.
+
+## Create new release branch
+
+The release branch should be named `release-M.N.x` where `M.N` is replaced with the major and minor versions of the new release.
+For example:
+
+```
+git checkout -b release-0.2.x
+```
+
+## Set new version in main Maven project
+
+Update the version for the new Maven project:
+
+```
+mvn versions:set -DnewVersion=X.Y.Z
+```
+
+Where `X.Y.Z` is the new version which is going to be released.
```

Comment:
should be M=X and Y=N? Why not using them, it could be confusing

Reply:
`X.Y.Z` is the normal placeholder for the version. But in the branch name it conflicets with the `.x` at the end. That is why different letters are used here and there.